collections/_infographics/emissions/emission-reduction-potential.md

```diff
@@ -3,12 +3,139 @@ layout:        infographic
 title:         "Potential of emission reduction pathways: Czechia"
 slug:          "emission-reduction-potential"
 redirect_from: "/emission-reduction-potential"
-published:     2022-07-26
+published:     2022-12-29
 weight:        65
 tags-scopes:   [ cz ]
 tags-topics:   [ emissions, policies ]
 caption:       "The infographic compares the amount of emissions that can be reduced if people radically change their consumption habits with the amount that can be cut via system changes implemented by the government. The highest potential is in the change of the Czech energy generation mix: shut down the lignite power plants and replace them with energy sources that have lower emission intensity."
 data-our:      "https://docs.google.com/spreadsheets/d/1Mfua6djrufHgLh2MjNLxkJd7NxtaK1vVX2m8xLPGZLg/edit?usp=sharing"
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/potencial-zpusobu-snizeni-emisi" %}
+## How to read this graph?
+
+This graph compares indicative estimates of emissions savings in different areas of the company's operations. In the left-hand column, we show the potential for individual savings if a large part of the company were to get involved and collectively halve total consumption in a given category. This emission reduction factor is chosen symbolically; such targets would be very challenging to achieve (and vary in difficulty across categories, see below). For example, halving the distance flown would only save about 0.8% of total emissions compared to 2018.
+
+The right-hand column shows the potential for savings that are not directly related to individual consumption and require a more systemic approach to achieve. For example, planting 10 million extra trees would only save about 0.2% of total emissions in 2030 compared to 2018. In contrast, transforming the energy sector would deliver 50-100 times higher emissions savings.
+
+## Background to the section: What people can influence through their consumption
+
+In this section, we explain in more detail the different estimates and assumptions we make. The exact derivation of each figure can be found in our data table.
+
+### We will drive half the distance in private cars
+
+The largest group of Czech drivers drive between <!--[-->5 and 10 thousand km<!--](https://www.opojisteni.cz/pojistny-trh/pojistne-produkty/pruzkum-pojistovny-direct-jak-cesi-jezdi/c:11746/)--> per year and passenger cars are responsible for a total of 11.9 million tonnes of <glossary id="co2eq">CO<sub>2</sub>eq</glossary> emissions in the Czech Republic. **If the total annual mileage were halved, about 5.9 million tonnes of CO<sub>2</sub> emissions would be saved annually.**
```

Comment:
Unable to find this link in English. Not sure if there is an equivalent to https://www.opojisteni.cz/pojistny-trh/pojistne-produkty/pruzkum-pojistovny-direct-jak-cesi-jezdi/c:11746/